panoramix/templates/panoramix/dashboard.html

```diff
@@ -66,7 +65,7 @@ <h4 class="modal-title" id="myModalLabel">Dashboard CSS</h4>
       </div>
     </div>
 </div>
-<div class="gridster content_fluid">
+<div class="gridster content_fluid" style="visibility: hidden;">
```

Comment:
css rule instead of inline style! why is this necessary? and is `visibility: hidden;` what you want or `display: none;`?